### DISCUSSION TITLE
How are envelopes used in VDA5050?

### DISCUSSION THREAD

**Discussion by ajithcodesit** [VDA5050/VDA5050#311] [Q&A]
The envelope is defined in the factsheet but there is no field in the state structure to let the master control know which envelope is being used by the vehicle in the current moment.

So my question here is, how is the envelopes used by the vehicle and the master control? There seems to be very little information mentioned in the VDA5050 v2.0.0 document. Envelopes are mentioned being used with pivot points in page 10 other than that no discussion on it.

**Comment by MaximilianPenninger** [comment]
I was part of WG01 as the factsheet was defined for release 2.0. Please be aware, that the factsheet is not just meant to be transfered live from an an AGV to MC, but that a factsheet is also planned to be used for plant-planning, agv-comparison while RFQ-processes and so on.

The idea was, that a MC or a plant-simulation is able to work with correct kinds of envelopes. As the envelope-definiton in the factsheet is very open, the agv manufacturer has the possibility to offer a wide range of envelopes for different purposes. Some examples:

- envelope of the agv itself
- envelope(s) including safety field(s)
- envelope(s) in different heights
- envelope including load(s)
- ....
